### PR TITLE
chore(e2e tests): support some e2e tests on circle ci

### PR DESCRIPTION
**Pull request by sjelin**
Sadly, x86 emulation is impossible (no hardware extensions on circle) and arm is
unworkably slow, so no android tests.  See
https://github.com/angular/webdriver-manager/issues/167 for ideas about work
arounds

Removed travis tests since circle tests are cooler anyway

Made some timeouts longer because I guess circle is slow.  Also improved some
an error path in downloader because it was a problem while debugging.

Sample output: https://circleci.com/gh/sjelin/webdriver-manager/31

Once this is in we'll just have to turn on circle for this repo

Closes https://github.com/angular/webdriver-manager/issues/165

### CODE REVIEW

**Comment by cnishina** [comment]
Are we using Circle CI and no longer using Travis?

**Comment by sjelin** [comment]
On this repo I think it'd be the right decision.  The circle tests are a superset of the travis tests.

**Comment by sjelin** [comment]
Rebased, and you can see the circle CI check in the normal place now!

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+machine:
+  environment:
+    # Fix issue with selenium-server in containers.
+    # See http://github.com/SeleniumHQ/docker-selenium/issues/87
+    DBUS_SESSION_BUS_ADDRESS: /dev/null
+  post:
```
Comment (cnishina) [inline circle.yml]: I am not sure if we need this if we are using node version 6.

File: package.json [hunk 2]
```diff
@@ -64,5 +64,8 @@
     "run-sequence": "^1.1.5",
     "selenium-webdriver": "2.53.1",
     "typescript": "^2.1.1"
+  },
+  "engines": {
+    "node": ">=6.x.6"
```
Comment (cnishina) [inline package.json]: What version should we use? What version is 6.x.6? Should we use >= 6.0.x?

Comment (sjelin) [inline package.json]: I coped this from somewhere.  I think `>= 6.0.x` is equivalent to `>= 6.0.0`

File: circle.yml (hunk 1 above)
Comment (sjelin) [inline circle.yml]: I'll remove it and see!

**Comment by sjelin** [comment]
All comments addressed

**Pull request merged**